### PR TITLE
DS-5351 GDPR Consent bugs and changes

### PR DESCRIPTION
**Pull request by ghost**
You can see HTT [here](https://github.com/goalgorilla/open_social/pull/868).

### CODE REVIEW

File: src/Entity/UserConsentInterface.php [hunk 1]
```diff
@@ -13,6 +13,21 @@
  */
 interface UserConsentInterface extends ContentEntityInterface, EntityChangedInterface, EntityOwnerInterface {
 
+  /**
+   * The user only visits agreement page.
+   */
+  const STATE_UNDECIED = 0;
```
Comment (bramtenhove) [inline src/Entity/UserConsentInterface.php]: @OH17L this is a typo. Should be `STATE_UNDECIDED`

File: src/Entity/UserConsentInterface.php [hunk 2]
```diff
@@ -13,6 +13,21 @@
...
+
+  /**
+   * The user submitted agreement form but do not set checkbox.
+   */
+  const STATE_NOT_AGREE = 1;
+
+  /**
+   * The user has given consent on the current version of data policy.
+   */
+  const STATE_AGRRE = 2;
```
Comment (bramtenhove) [inline src/Entity/UserConsentInterface.php]: @OH17L this is a typo. Should be `STATE_AGREE`

**Comment by bramtenhove** [comment]
I'm just adding a few comments, sorry if you already noticed it :)

File: src/Entity/UserConsentInterface.php (hunk 2 above)
Comment (OH17L) [reply src/Entity/UserConsentInterface.php]: @bramtenhove, you are right. I will fix it.

File: src/Entity/UserConsentInterface.php (hunk 1 above)
Comment (OH17L) [reply src/Entity/UserConsentInterface.php]: @bramtenhove, you are right. I will fix it.

**Pull request merged**